pom.xml

```diff
@@ -67,7 +67,7 @@
         <jdk-version>1.${short-jdk-version}</jdk-version>
 
         <!-- versions -->
-        <zookeeper-version>3.6.0</zookeeper-version>
+        <zookeeper-version>3.6.1</zookeeper-version>
```

Comment:
Mandatory in order to use:

`new ZooKeeperAdmin(connectString, sessionTimeout, watcher, canBeReadOnly, zkClientConfig)`